pkg/sqlutil/pg/connection_test.go

```diff
@@ -0,0 +1,31 @@
+//go:build db
+
+package pg
+
+import (
+	"testing"
+
+	"github.com/google/uuid"
+	_ "github.com/jackc/pgx/v4/stdlib"
+	"github.com/jmoiron/sqlx"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_disallowReplica(t *testing.T) {
+	db, err := sqlx.Open(string(DriverTxWrappedPostgres), uuid.New().String())
```

Comment:
Is this test working already?  Or is the plan for it to work once the PR that moves txdb into common is merged?

(As I understand it, this shouldn't work unless `sql.Register()` is called first, to register the txdb driver and implement it.)

Reply:
It is moved from core so it is a working test, but it is behind a build tag so it is not enabled in CI currently (or when invoking plain `go test`). I don't recall having a particular plan, but we can activate it with your PR